megazords/ios/MozillaAppServices.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 52;
+	objectVersion = 51;
```

Comment:
This, I don't know what it is, some Xcode mystery...

Reply:
I _think_ this means older Xcode might refuse to load that project, but IMO that's fine 🤷